.github/labeler.yml

```
@@ -4,22 +4,10 @@
 
 Python:
   - 'python/**'
-  - 'notebooks/**'
-  
-librmm:
-  - 'cpp/**'
 
 CMake:
   - '**/CMakeLists.txt'
   - '**/cmake/**'
-  
-Java:
-  - 'java/**'
-  
-Ops:
-  - '.github/**'
-  - 'ci/**'
+
+conda:
   - 'conda/**'
```

I'd add gpuCI to match cuDF and other repos
```suggestion
  - 'conda/**'

gpuCI:
  - 'ci/**'
```